api/queue/work/model_service_deployment.go

```diff
@@ -185,7 +185,7 @@ func (depl *ModelServiceDeployment) generateModelOptions(ctx context.Context, mo
 	modelOpt := &models.ModelOption{}
 	switch model.Type {
 	case models.ModelTypePyFunc:
-		imageRef, err := depl.ImageBuilder.BuildImage(ctx, model.Project, model, version)
+		imageRef, err := depl.ImageBuilder.BuildImage(ctx, model.Project, model, version, version.ImageBuilderResourceRequest)
```

Comment:
the `buildimage` method isn't updated to pass the `ImageBuilderResourceRequest`.

however is this required? version is already passed